_data/documentation/0_13_0.yaml

```diff
@@ -0,0 +1,38 @@
+docs:
```

Comment:
This is an example of the YAML which is used to drive the documentation index page. This is/will be generated from the `mvn` build in https://github.com/kroxylicious/kroxylicious, so the metadata about a piece of documentation lives beside that documentation.